index.js

```diff
@@ -31,7 +31,22 @@ function pipeLogsToTerminal(config) {
     subject(...args);
   });
 
-  Cypress.on('log:added', options => {
+  Cypress.on('command:end', async ({attributes}) => {
```

Comment:
This is fire only when the command is successful. The response is appended to the last the log.